modules/integration/src/test/scala/scala/cli/integration/RunTestDefinitions.scala

```diff
@@ -488,34 +506,36 @@ abstract class RunTestDefinitions(val scalaVersionOpt: Option[String])
       val expectedLines =
         if (actualScalaVersion.startsWith("2.12."))
           s"""Exception in thread "main" java.lang.ExceptionInInitializerError
-             |${tab}at throws.main(throws.sc)
+             |${tab}at throws_sc$$.main(throws.sc:23)
+             |${tab}at throws_sc.main(throws.sc)
              |Caused by: java.lang.Exception: Caught exception during processing
              |${tab}at throws$$.<init>(throws.sc:6)
              |${tab}at throws$$.<clinit>(throws.sc)
-             |$tab... 1 more
+             |$tab... 2 more
              |Caused by: java.lang.RuntimeException: nope
              |${tab}at scala.sys.package$$.error(package.scala:30)
              |${tab}at throws$$.something(throws.sc:2)
              |${tab}at throws$$.<init>(throws.sc:3)
-             |$tab... 2 more
-             |""".stripMargin.linesIterator.toVector
+             |$tab... 3 more""".stripMargin.linesIterator.toVector
         else
           s"""Exception in thread "main" java.lang.ExceptionInInitializerError
-             |${tab}at throws.main(throws.sc)
+             |${tab}at throws_sc$$.main(throws.sc:23)
+             |${tab}at throws_sc.main(throws.sc)
              |Caused by: java.lang.Exception: Caught exception during processing
              |${tab}at throws$$.<clinit>(throws.sc:6)
-             |$tab... 1 more
+             |$tab... 2 more
              |Caused by: java.lang.RuntimeException: nope
              |${tab}at scala.sys.package$$.error(package.scala:27)
              |${tab}at throws$$.something(throws.sc:2)
              |${tab}at throws$$.<clinit>(throws.sc:3)
-             |$tab... 1 more
+             |$tab... 2 more
              |""".stripMargin.linesIterator.toVector
       if (exceptionLines != expectedLines) {
-        pprint.log(exceptionLines)
-        pprint.log(expectedLines)
+        println(exceptionLines.mkString("\n"))
+        println(expectedLines)
       }
-      expect(exceptionLines == expectedLines)
+      expect(exceptionLines.length == expectedLines.length)
+      for { i <- Range(0, exceptionLines.length) } expect(exceptionLines(i) == expectedLines(i))
```

Comment:
NIT, more idiomatic way of doing this:
```suggestion
      for (i <- 0 until exceptionLines.length) expect(exceptionLines(i) == expectedLines(i))
```